src/Engine/Trigger/TriggerOnce.cpp

```diff
@@ -0,0 +1,69 @@
+#include "TriggerOnce.hpp"
+
+TriggerOnce::TriggerOnce() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'TriggerOnce::startActive' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)